pkgs/by-name/xx/xxh/package.nix

```
@@ -5,21 +5,25 @@
   openssh,
   nixosTests,
 }:
-
 python3.pkgs.buildPythonApplication rec {
   pname = "xxh";
-  version = "0.8.12";
-  format = "setuptools";
+  version = "0.8.14";
+  format = "pyproject";
 
   disabled = python3.pkgs.pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     tag = version;
-    hash = "sha256-3/AU2o72X7FE11NSXC6m9fFhmjzEDZ+OpTXg8yvv62A=";
+    hash = "sha256-Y1yTn0lZemQgWsW9wlW+aNndyTXGo46PCbCl0TGYspQ=";
   };
 
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+  ];
+
   propagatedBuildInputs = [
```

```suggestion
  build-system = [
    python3.pkgs.setuptools
  ];

  dependencies = [
```
Wheel is not required